src/behaviors/Behavior.js

```diff
@@ -6,7 +6,7 @@ var Base = require('../lib/Base');
 
 var Column = require('./Column');
 var dialogs = require('../dialogs');
-var CellProvider = require('../lib/CellProvider');
+var CellProvider = require('../cellRenderers/CellProvider');
```

Comment:
As it will reorganize the way cell renderers are defined, I think for the better, it might be worth doing now. When are you thinking we would do it?